### ISSUE TITLE
Request: Clarify and increase menu options

### ISSUE THREAD

**Issue by egabrum** [tim-we/tabs-aside#93]
I don't use the drop-down menu from the icon in the Extension bar. I find there are not enough options and, for the ones the are there, it's not too clear for me what they do from just the title:

"Tabs aside", : I use many extensions and software. I don't always remember what "Tabs aside" exactly does. The language is too succinct.
"Save tabs as session": Likewise. Just save? Or set aside too? (the language opposition with "Tabs aside" is not sufficient) Which tabs?

I'd suggest:

"Set all tabs aside in..." > (new session/existing sessions)
"Save all tabs in..." > (new session/existing sessions)
"Set selected tabs aside in..." > (new session/existing sessions)
"Save selected tabs in..." > (new session/existing sessions)

**Comment by tim-we** [comment]
Yes I agree the language used could be a little bit better. Did you notice the tooltips when you hover the menu entries?

**Comment by egabrum** [comment]
Yes! And I appreciate them. But my suggestion is to make it even more user friendly.
